tensorflow/python/ops/math_ops.py

```diff
@@ -13,7 +13,26 @@
 # limitations under the License.
 # ==============================================================================
 
-"""## Arithmetic Operators
+"""Note: Elementwise binary operations in TensorFlow follow [numpy-style
+broadcasting](http://docs.scipy.org/doc/numpy/user/basics.broadcasting.html).
+Let us take an example of `tf.mul()` which falls in the above mentioned
+category, to clarify broadcasting and also give example usage of
+`tf.expand_dims()` when broadcasting is desired.
+
+Ex: Let us say that we have a batch of images in the tensor `image_batch` which
+is of size `b x h x w x c`, and a mask tensor `image_mask` of size `h x w` that
+we need to apply to all the images in the batch across all the channels. We thus
```

Comment:
Do you think it's necessary to have additional text here?  Why isn't the link to the numpy page enough?  Thoughts @martinwicke  ?